x-pack/platform/plugins/shared/lens/public/editor_frame_service/editor_frame/config_panel/layer_panel.test.tsx

```diff
@@ -689,7 +689,8 @@ describe('LayerPanel', () => {
       );
     });
 
-    it('should determine if the datasource supports dropping of a field onto a pre-filled dimension', async () => {
+    // TODO: Doesn't work because of emotion babel css prop preset
+    it.skip('should determine if the datasource supports dropping of a field onto a pre-filled dimension', async () => {
```

Comment:
Pinging @elastic/kibana-visualizations to help with this or file a followup issue :)

Reply:
Created https://github.com/elastic/kibana/issues/217406 to take a look.